### PR TITLE
Add notify_events notification component documentation

### PR DESCRIPTION
**Pull request by papajojo**
## Proposed change

Adding a description for new platform for notify component.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [#36049](https://github.com/home-assistant/core/pull/36049)
- Link to parent pull request in the Brands repository: [#1651](https://github.com/home-assistant/brands/pull/1651)
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/notify_events.markdown [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: "Notify.Events"
+description: "Instructions on how to integrate Notify.Events service with your Home Assistant notifications."
+ha_release: "0.110.1"
```
Comment (klaasnicolaas) [inline source/_integrations/notify_events.markdown]:
```suggestion
ha_release: 0.111
```

File: source/_integrations/notify_events.markdown [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+ha_category: Notifications
```
Comment (klaasnicolaas) [inline source/_integrations/notify_events.markdown]:
```suggestion
ha_category:
  - Notifications
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by papajojo** [comment]
Thanks for the text tweaks btw!

**Review by frenck — APPROVED** [review]

**Comment by papajojo** [comment]
Hi @frenck! Could you please tell me when approximately is our component going to release? Asking because we are planning to send information to our subscribers about that integration. Thanks!

**Comment by klaasnicolaas** [comment]
June 24 is the beta and release on July 1

**Pull request merged**